backend/helper_zero/models.py

```diff
@@ -57,7 +37,8 @@ class Donation(models.Model):
   item_type = models.CharField(max_length=120, null=True)
   amount = models.PositiveIntegerField()
   created_at = models.DateTimeField()
-  donation_time = models.DateTimeField()
+  donation_time_start = models.DateTimeField()
+  donation_time_end = models.DateTimeField(null=True)
```

Comment:
Is this to enforce a mandatory start time but make donation end times optional?

Reply:
ah no, both are required. the reason we need to add `null=true` to certain values is because of existing rows we have in our local organizations table. from running the migration, i decided to rename donation_time to donation_time_end - django forces you to set a default value when you add new required rows if you're changing an existing column, because that's how they choose to handle database conflicts.

Reply:
it sounds like this is a byproduct of the fact that you ran a local migration or entered tables into your sqlite db that  would cause a db conflict.

i don't think we should be forced to add `null=True` to fields just to satisfy these constraints; consider deleting your db and removing null=True and just rerunning your migration? let me know if this makes sense.

Reply:
By that same logic we shouldn't have added null=True for all the others - what I can do is delete my DB, get rid of the other null=True, and then re-run. However it would also mean everyone else has to delete everything in their DB currently.